Documentation/installation/servicemesh/ingress.rst

```diff
@@ -24,6 +24,32 @@ an existing K8s cluster with Cilium installed.
 
 .. include:: installation.rst
 
+Supported Ingress Annotations
+#############################
+
+.. list-table:: 
```

Comment:
@qmonnet need your help to confirm if we can use this format in docs. Other tables that we have in docs (e.g dev_setup.rst) are following normal md format.

Reply:
We're not usually using it, even though the Helm reference does. But I think it's simply because nobody tried it before, I don't think we rejected the possibility. If it builds and renders fine, I've got no objection. It's less intuitive to read as a regular table, but we advantage of not having to deal with all the underlines and borders in the source file is real. There are few enough columns that it still reads well even as a list in the RST, in my opinion, so fine by me

Reply:
Thanks, I have verified this locally, the table looks good to me :100: 

<img width="580" alt="image" src="https://user-images.githubusercontent.com/9019229/185607771-8bb4aeb1-0fd5-42e3-8b5d-2f7c8db1ed9e.png">

Reply:
we can try to increase the width of first column, but we can do it whenever we add new annotations, which I am sure there are more coming.